resources/assets/logisticspipes/lang/en_us.lang

```diff
@@ -502,3 +502,15 @@ gui.popup.managechannel.deletedialog.no=No
 itemstackinfo.lprouteditem=%cRED Item with stored LogisticsPipes routing information
 itemstackinfo.lproutediteminfo=%cYELLOW Drop to remove routing information and get original items back
 itemstackinfo.lprouteditemtype=Stored item
+
+book.quickstart.title=Logistics Pipes Quickstart Guide
+book.quickstart.1="It's been a long time. How have you been?"\nFirst off, thanks for playing Logistics Pipes. A lot of changes have been made since the last release of LP in Minecraft 1.7.10. We'll try to explain some of these in this book. Read the next pages, if you'd like to know them.
+book.quickstart.2=I hope you already have your Logistics Power Junction set up and crafted your first Basic Logistics Pipes :)\nIf not, go for it.\nYou might have recognized that all other types of pipes and modules now need something called a Logistics Programmer.
+book.quickstart.3=The Logistics Programmer is an item that wont be used up in any crafting recipes, but you need to set it up first. That is to be done with a Logistics Program Compiler. I know, I know, it sounds complicated. But it really isn't. Craft both of these and a Logistics Disk.
+book.quickstart.4=Now you place the Logistics Program Compiler (further just compiler) next to a Logistics Basic Pipe that is - by other pipes or directly - connected to the Power Junction, because the compiler needs power and will take that from the LP network.
+book.quickstart.5=The compiler can be used to research new tiers, pipes, modules and upgrades. Insert the Logistics Disk on the top left. Then you are able to unlock different tiers. Let's start with 'Basic'. Researching takes some time, so relax and get some coffee.
+book.quickstart.6=After unlocking a tier, you should be able to see the different pipe, module or upgrade names on the right. You can now compile each of these, which takes a shorter amount of time. Compiling makes those programs flashable on the Logistics Programmer.
```

Comment:
"Compiling makes those programs flashable _onto / to_ the Logistics Programmer."?

Reply:
I think this is correct